lib/sober_swag/compiler/type.rb

```diff
@@ -196,13 +203,15 @@ def flatten_oneofs_hash(object)
 
       def path_schema_stub
         @path_schema_stub ||=
-          object_schema[:properties].map do |k, v|
+          make_object_schema(metadata_keys: METADATA_KEYS | %i[style explode])[:properties].map do |k, v|
             # ensure_uncomplicated(k, v)
             {
               name: k,
-              schema: v.reject { |key, _| %i[required nullable].include?(key) },
-              required: object_schema[:required].include?(k) || false
-            }
+              schema: v.reject { |key, _| %i[required nullable explode style].include?(key) },
+              required: object_schema[:required].include?(k) || false,
+              style: v[:style],
+              explode: v[:explode]
+            }.reject { |_, v2| v2.nil? }
```

Comment:
@AnthonySuper is this where sober_swag is removing blank `""` values in an array that is an issue @connorshea 
 referenced in https://github.com/SonderMindOrg/sonder-rails/pull/3466#discussion_r482325028?

Reply:
@ap2322 I just cherry-picked the sober_swag 0.15 commits onto the religions branch, removed the `.constructor { |input| input.reject(&:blank?) })` hack, and it does actually return nil again!

So I guess it does fix that, neat.

Reply:
Cool, thanks for verifying!